src/tasks.py

```diff
@@ -63,30 +65,113 @@ def command(
     """
     input_files = get_input_files(pipe_result, input_files or [])
     output_files = []
-    base_command = ["<REPLACE_WITH_COMMAND>"]
-    base_command_string = " ".join(base_command)
+
+    ssh_analysis_task = LinuxSSHAnalysisTask()
```

Comment:
You first call the read_logs function with the input_files, this gives you a pd with all the data.
Then you call the brute_force_analysis function with that pd as input, this will give you the report. 
https://github.com/google/turbinia/blob/cedf5c65d3c81f96903ac89c0bfed7d68b9cf82f/turbinia/workers/analysis/ssh_analyzer.py#L376

The output of that analysis function is your outpout_file report.